### PR TITLE
feat: more friendly cli error message

### LINKED ISSUE #1211: `limactl validate` should provide help
### Description

This is user friendly:
```
$ ./bin/limactl
Lima: Linux virtual machines

Usage:
  limactl [command]

Examples:
  Start the default instance:
  $ limactl start

  Open a shell:
  $ lima

  Run a container:
  $ lima nerdctl run -d --name nginx -p 8080:80 nginx:alpine

  Stop the default instance:
  $ limactl stop

  See also example YAMLs: /home/user/share/doc/lima/examples

Available Commands:
  completion    Generate the autocompletion script for the specified shell
  copy          Copy files between host and guest
  delete        Delete an instance of Lima.
  disk          Lima disk management
  edit          Edit an instance of Lima
  factory-reset Factory reset an instance of Lima
  help          Help about any command
  info          Show diagnostic information
  list          List instances of Lima.
  prune         Prune garbage objects
  shell         Execute shell in Lima
  show-ssh      Show the ssh command line
  start         Start an instance of Lima
  stop          Stop an instance
  sudoers       Generate the content of the /etc/sudoers.d/lima file
  validate      Validate YAML files

Flags:
      --debug     debug mode
  -h, --help      help for limactl
  -v, --version   version for limactl

Use "limactl [command] --help" for more information about a command.
```

This is not:
```
$ ./bin/limactl validate
FATA[0000] requires at least 1 arg(s), only received 0
```

validate does take arguments
```
$ ./bin/limactl validate --help
Validate YAML files

Usage:
  limactl validate FILE.yaml [FILE.yaml, ...] [flags]

Flags:
  -h, --help   help for validate

Global Flags:
      --debug   debug mode
```

Note that this feels like I'm giving it an argument:
```
$ ./bin/limactl validate --debug
FATA[0000] requires at least 1 arg(s), only received 0
```

### PR DESCRIPTION
**Pull request by lobshunter**
This PR closes #1211, show more user friendly error message on argument error. The output is similar to nerdctl and docker cli.

It looks like below:
Screenshot 2023-02-28 at 19 01 06

### CODE REVIEW

File: cmd/limactl/main.go [hunk 1]
```diff
@@ -119,3 +119,18 @@ func handleExitCoder(err error) {
 		return
 	}
 }
+
+func WrapArgsError(argFn cobra.PositionalArgs) cobra.PositionalArgs {
```
Comment (AkihiroSuda) [inline cmd/limactl/main.go:123]: Please consider adding a comment line

**Review by AkihiroSuda — DISMISSED** [review]
Thanks

File: cmd/limactl/main.go (hunk 1 above)
Comment (lobshunter) [reply cmd/limactl/main.go:124]: Thanks for quick review, updated in 48ae695f7bda507f9209d2226c31c72959adb0cd.

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**